react/prop-types-typescript/README.md

```diff
@@ -0,0 +1,166 @@
+# Ratchet
```

Comment:
I think we should rewrite this README.md according to our rules of codemod README.
Especially that the commands in this file are not accurate anymore.